### DISCUSSION TITLE
Masonry Customization: Where can I change bootscore_category_badge() php

### DISCUSSION THREAD

**Discussion by paxaeterna** [bootscore/bootscore#747] [Q&A]
I enabled the masonry layout on my index.php for my blog. I want to change the way the category badge is styled and remove the link tag. This element pulls in the text from the posts tag you created and turns it into a clickable button. Where is this function defined so I may change it?

Section in the masonry php:
`<?php bootscore_category_badge(); ?>`

Rendered code:
`<p class="category-badge"><a href="https://dev.sitename.com/./news/" class="badge text-bg-light text-decoration-none">News</a></p>`

**Comment by paxaeterna** [comment]
Sorry, I found it! It is in inc > template.tags.php
Should I delete this question?

**Reply by crftwrk** [reply]
> Should I delete this question?

Leave it open, maybe someone else have the same question ;-)
